runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -449,7 +449,8 @@ J9::ClassEnv::enumerateFields(TR::Region& region, TR_OpaqueClassBlock * opaqueCl
             dataType = TR::Double;
             break;
             }
-         case 'L': 
+         case 'L':
+         case 'Q':
```

Comment:
Should this `Q` have a similar comment added?

Reply:
Sorry about the churn!  I inadvertently pushed my changes that time before I was ready.